Reference/Management/Services/SectionService/Index.md

```diff
@@ -1,9 +1,10 @@
 ---
 versionFrom: 8.0.0
+needsV8Update: "true"
```

Comment:
I think this should be `versionRemoved: 8.0.0` instead, is it sounds like it has been replaced by something else 🤔

Reply:
Yup, it no longer exists and is redundant. Will add the `versionRemoved` tag instead.